### DISCUSSION TITLE
Base class for collection fixture

### DISCUSSION THREAD

**Discussion by bulat-gab** [xunit/xunit#2681] [Question] [answered]
Basically, I'm trying to add an abstract BaseDatabaseFixture class to the DatabaseFixture class from this docs https://xunit.net/docs/shared-context#collection-fixture, collection fixture section. The BaseDatabaseFixture class will be abstract and set up some common things, like connect to the db, initialize a common MongoDB collections.
Then, the BaseDatabaseFixture class will be subclassed in other test projects, with some project specific code.

I'm getting an error: `The following constructor parameters did not have matching fixture data: BaseDatabaseFixture`
Neither my BaseDatabaseFixture class nor DabaseFixture classes have constructor parameters

Is it possible to achieve?

Full error message:
`System.AggregateException : One or more errors occurred. (Object reference not set to an instance of an object.) (The following constructor parameters did not have matching fixture data: BaseDatabaseFixture baseDatabaseFixture)
---- System.NullReferenceException : Object reference not set to an instance of an object.
---- The following constructor parameters did not have matching fixture data: BaseDatabaseFixture baseDatabaseFixture`

**Comment by bradwilson** [comment, marked as answer]
Can you provide a repro project?

**Reply by bradwilson** [reply]
Marking as answered. Please open an issue with a repro project. Thanks!
